pkg/debug/recover-tpm/main.go

```diff
@@ -466,7 +466,7 @@ func main() {
 			os.Exit(1)
 		}
 
-		encVkeyWire, err := getVualtKeyWireFormat(encVkey, digest256)
+		encVkeyWire, err := getVaultKeyWireFormat(encVkey, digest256)
```

Comment:
mind to squash this commit ?

Reply:
yes, I will move it to pkg/recovertpm and take care of it.